rpm-preamble.file

```diff
@@ -231,3 +231,24 @@
 %else
 %define strip strip
 %endif
+
+# Define a macro for adding dependencies
+%define addDependency \
+mkdir -p %i/etc/profile.d \
```

Comment:
@vkuznet , please use %{pkginstroot} instead of %{i}. Note that cmsBuild replaces %i with %{pkginstroot} in the spec files but I am not sure if it does it for rpm-preable.file or not. so better to use the full macro ie. %{pkginstroot}